src/main/kotlin/blackjack/flow/BlackJackGame.kt

```diff
@@ -0,0 +1,39 @@
+package blackjack.flow
+
+import blackjack.card.Deck
+import blackjack.participant.Dealer
+import blackjack.participant.Participants
+import blackjack.participant.Player
+import blackjack.participant.Players
+import blackjack.view.OutputView
+import blackjack.view.policy.InputMoreCardPolicy
+
+data class BlackJackGame(
+    val dealer: Dealer,
+    val gamePlayers: Players,
```

Comment:
```suggestion
    val participants: Participants
```

play를 통해서 Participants를 반환하여, 이에 대한 결과를 다시 추출하는 부분보다, BlackJackGame을 만들때부터 dealer와 gamePlayer를 Participants로 묶어주는 건 어떨까요?